core/Cargo.toml

```diff
@@ -8,19 +8,22 @@ license = "BSD-3-Clause"
 
 [lib]
 name = "lockbook_core"
-crate-type = ["lib", "staticlib", "cdylib"]
+crate-type = ["lib", "staticlib"]
 
 [dependencies]
-reqwest = "0.9.24"
+reqwest = { version = "0.10.4", features = ["blocking", "json"] }
 base64 = "0.11.0"
 serde = { version = "1.0", features = ["derive"] }
 serde_json = "1.0.44"
 uuid = { version = "0.8.1", features = ["v4"] }
-sled = { version = "0.31.0" }
```

Comment:
Will we always need this change?

Reply:
No once it's resolved and he bumps the version it goes back, I'll add a TODO